<?xml version="1.0"?>
<launch>
  <!-- First launch the standard Scout Mini simulation -->
  <include file="$(find scout_gazebo_sim)/launch/scout_mini_empty_world.launch" />
  
  <!-- Launch our waypoint follower node with a delay -->
  <node pkg="scout_mini_nav" type="waypoint_follower_node" name="waypoint_follower_node" 
        output="screen" launch-prefix="bash -c 'sleep 15; $0 $@'">
    <param name="linear_speed" value="0.3" />
    <param name="angular_speed" value="0.5" />
    <param name="waypoint_reached_distance" value="0.3" />
    <param name="angle_tolerance" value="0.1" />
  </node>
</launch>
